igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/service/TravisService.java

```diff
@@ -444,26 +443,20 @@ private Optional<String> getAndCacheJobLog(int jobId) {
         travisCache.setJobLog(groupKey, jobId, v3Log.getContent());
         return Optional.of(v3Log.getContent());
       }
-    } catch (RetrofitError e) {
-      if (e.getBody() != null) {
-        try {
-          Map body = (Map) e.getBodyAs(HashMap.class);
-          if ("log_expired".equals(body.get("error_type"))) {
-            log.info(
-                "{}: The log for job id {} has expired and the corresponding build was ignored",
-                groupKey,
-                jobId);
-          } else {
-            log.warn(
-                "{}: Could not get log for job id {}. Error from Travis:\n{}",
-                groupKey,
-                jobId,
-                body);
-          }
-        } catch (RuntimeException ex) {
-          log.warn("{}: Could not parse original error message from Travis", groupKey, ex);
+    } catch (SpinnakerServerException e) {
+      if (e.getResponseBody() != null) {
```

Comment:
Let's call `e.getResponseBody()` once and check the resulting variable for null.